taxonomy-editor-frontend/src/components/Loader.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
```

Comment:
I think it is ok to replace the 'loading...' message with a spinner. But for the moment I'd say we could just use the `CircularProgress` directly, without the need for a wrapper, this gives us more control on the spacing, size, etc. Please instead of importing this component, use the `CircularProgress`.